ccd-definition/ComplexTypes/CareSupervision/CaseManagementOrder/DEPRECATED/CaseManagementOrder.json

```diff
@@ -75,5 +75,13 @@
     "FieldTypeParameter": "CMOStatus",
     "ElementLabel": "Is this ready to be sent to the judge?",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "nextHearing",
+    "FieldType": "NextHearing",
```

Comment:
I think that this field here is the only one that really needs reverting, do we wanna do just this one or all of them for safety?